### PR TITLE
WIP: 955 video block

### PR DESCRIPTION
**Pull request by hippogriffic**

### CODE REVIEW

**Comment by hippogriffic** [comment]
need to fix some mobile styles

File: django-verdant/rca/static/rca/css/streamfield-blocks/video-block.less [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+        margin: 0;
+        // Reduce width to force line-breaks
+        max-width: 140px;
+
+        @media @media-desktop-small {
+            line-height: 0.75;
+        }
+
+        .playing & {
+            display: none;
```
Comment (helenb) [inline django-verdant/rca/static/rca/css/streamfield-blocks/video-block.less]: Ha! that was easier than I was expecting... thought we'd need to listen to a js event

File: django-verdant/rca/templates/rca/blocks/video_block.html [hunk 2]
```diff
@@ -1,7 +1,18 @@
 {% load wagtailembeds_tags wagtailimages_tags %}
 
-{{ value.title }}
+<div class="videoembed vimeo">
+    <h2 class="block-video__title homepage-h2">{{ value.title }}</h2>
 
-{{ value.video.url|embed }}
+    <button class="playpause play block-video__play-button">
+        <svg class="block-video__play-icon">
```
Comment (helenb) [inline django-verdant/rca/templates/rca/blocks/video_block.html]: Double check accessibility - does it need a title?

File: django-verdant/rca/static/rca/css/streamfield-blocks/video-block.less [hunk 3]
```diff
@@ -0,0 +1,55 @@
...
+    &__play-button {
+        transform: translateY(-50%);
+        border: 0;
+        padding: 0;
+        display: flex;
+        justify-content: center;
+        align-items: center;
+
+        .videoembed & {
+            top: 50% !important;
```
Comment (helenb) [inline django-verdant/rca/static/rca/css/streamfield-blocks/video-block.less]: assume the !important is needed to override an inline style?

**Comment by helenb** [comment]
All looks good so far - just a few minor comments

File: django-verdant/rca/templates/rca/blocks/video_block.html (hunk 2 above)
Comment (hippogriffic) [reply django-verdant/rca/templates/rca/blocks/video_block.html]: added

**Comment by hippogriffic** [comment]
changes made, updated mobile text style

**Comment by helenb** [comment]
Looks good - happy for you to merge.

**Pull request merged**